python/Dockerfile

```diff
@@ -55,69 +55,25 @@ USER dependabot
 ENV DEPENDABOT_NATIVE_HELPERS_PATH="/opt"
 RUN git -c advice.detachedHead=false clone https://github.com/pyenv/pyenv.git --branch $PYENV_VERSION --single-branch --depth=1 /usr/local/.pyenv
 
-FROM python-core as python-3.10
-RUN pyenv install $PY_3_10 \
-  && bash /opt/python/helpers/build $PY_3_10 \
+FROM python-core as python-3.8
```

Comment:
This was a simple re-ordering as `3.11` was already at the bottom, so easier to go oldest to newest, rather than newest to oldest, and then back to newest.